R/shiny_vector_filter_character.R

```diff
@@ -19,12 +19,22 @@ shiny_vector_filter.character <- function(data, inputId, ...) {
   module_return <- shiny::reactiveValues(code = TRUE, mask = TRUE)
   
   output$ui <- shiny::renderUI({
+    
     filter_log("updating ui", verbose = verbose)
-    proportionSelectInput(ns("param"), NULL,
-                          vec = x,
-                          selected = shiny::isolate(input$param) %||% c(),
-                          multiple = TRUE,
-                          width = "100%")
+    
+    if (map(x(), is.empty) %>% reduce(`&`)) {
+      shiny::div(style = "opacity: 0.5;",
+                 p(width = "100%", 
+                   align = "center", 
+                   "Input is blank"))
```

Comment:
`"Column is blank"))`

Reply:
Since the users don't know what an "input" is, could we use the work "Column" or "Variable" here?

Reply:
Or better yet, "Variable only contains blank values"??

Reply:
Great idea! I was just focused on getting it working - love that